spinalcordtoolbox/deepseg/models.py

```diff
@@ -76,6 +76,14 @@
         "description": "Grey/white matter seg on exvivo human T2w.",
         "contrasts": ["t2"],
         "default": False,
+    },
+    "model_7t_multiclass_gm_sc_unet2d": {
+        "url": [
```

Comment:
Could you also update the URL to the new `.zip`? 

https://github.com/ivadomed/model_seg_gm-wm_t2star_7t_unet3d-multiclass/archive/r20210930.zip